lib/ohai/plugins/fips.rb

```diff
@@ -30,6 +30,6 @@
     fips Mash.new
 
     require "openssl" unless defined?(OpenSSL)
-    fips["kernel"] = { "enabled" => OpenSSL::OPENSSL_FIPS }
+    fips["kernel"] = { "enabled" => OpenSSL.fips_mode }
```

Comment:
Oh, I see this was reverted in ab6180be02712512a3a8309e75424dc56520b936. OpenSSL v3.0 may not have `fips_mode`.